rmw_fastrtps_cpp/src/namespace_prefix.cpp

```diff
@@ -29,13 +29,25 @@ const std::vector<std::string> _ros_prefixes =
 }  // extern "C"
 
 /// Return the ROS specific prefix if it exists, otherwise "".
-std::string
+const std::string
```

Comment:
Since the return type is a value and not a pointer or reference adding `const` doesn't make any sense.

Same below.

Reply:
That's correct (checked with the profiler). I've fixed that and uploaded the change.